### PR TITLE
Updated docs for cloudTrail dashboard

### PR DESCRIPTION
**Pull request by himsharma01**
## Purpose of this pull request

This pull request... 

## Select the type of change

- [ ] Minor Changes - Typos, formatting, slight revisions
- [x] Update Content - Revisions, updating sections
- [ ] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - .clabot, version updates, maintenance, dependencies, new packages for the site (Docusaurus, Gatsby, React, etc.)

### CODE REVIEW

File: docs/integrations/amazon-aws/application-load-balancer.md [hunk 1]
```diff
@@ -207,3 +207,15 @@ Use this dashboard to:
...
+
+The **AWS Application Load Balancer - CloudTrail Audit** dashboard provides a comprehensive overview of AWS Application Load Balancer activities through CloudTrail audit logs. It visualizes successful and failed events globally, event trends, error details, and user activities, offering insights into load balancer performance, security, and usage patterns.
+
+Use this dashboard to:
+* Monitor the geographical distribution of successful and failed load balancer events, allowing for quick identification of regions with high activity or potential issues.
+* Track the overall success rate of load balancer events and analyze trends over time, helping to identify any sudden changes or patterns in performance.
+* Investigate specific error events, including their details, frequency, and associated users, enabling faster troubleshooting and resolution of issues.
+* Identify the most common error types and the users experiencing the highest failure rates, facilitating targeted improvements and user support.
+
+<img src={useBaseUrl('img/integrations/amazon-aws/AWS-Application_Load_Balancer-Threat_Intel.png')} alt="AWS Application Load Balancer dashboard" />
```
Comment (jpipkin1) [inline docs/integrations/amazon-aws/application-load-balancer.md:221]: @himsharma01, this is not the correct screen image.

**Review by jpipkin1 — CHANGES_REQUESTED** [review]
@himsharma01, you need to correct some of the screen images.

**Review by JV0812 — APPROVED** [review]
LGTM

**Review by jpipkin1 — APPROVED** [review]
Thanks for updating the screen images, @himsharma01. It looks good now.

**Pull request merged**